### ISSUE TITLE
Building Anaconda Mandatory Packages - 6.1 Testing Python 3.6 in virtual environments

### ISSUE THREAD

**Issue by kingaj12** [linux-on-ibm-z/docs#37]
Successful up to this point, but now it seems that conda is a no-op:

    [linux1@linuxoneajk ~]$ which conda
    ~/miniconda3/bin/conda
    [linux1@linuxoneajk ~]$ conda --version
    [linux1@linuxoneajk ~]$

**Comment by cwsolee** [comment]
Hi Alan, Anaconda isn't one of the package that we've been keeping current. I assumed u started from here https://github.com/linux-on-ibm-z/docs/wiki/Building-Anaconda? The version is quite out of date now and we haven't seen enough activities to keep it current. Are u still trying to build this old version?

**Comment by kingaj12** [comment]
Yes, I did start there.  I actually solved this problem by creating a new environment.  So far I am making good progress, and I am documenting my victories.  I will continue working on this (from time to time).  I think that as LinuxONE gets more subscriptions you will see more demand for Anaconda package management.
